### ISSUE TITLE
Dryrun rename fails for attached file (and most everything i have, apparently)

### ISSUE THREAD

**Issue by automatonic** [dylansmith/node-exif-renamer#5]
`exif-renamer 1ssmzekd562-5mq5af1fjagtm.jpg -d`
yields
```
ERROR: TypeError: Invalid date
ERROR: Error: No Exif segment found in the given image.
```
![1ssmzekd562-5mq5af1fjagtm](https://cloud.githubusercontent.com/assets/139759/5902779/2be4ff00-a532-11e4-93f6-a2cf087996c6.jpg)

**Comment by dylansmith** [comment]
Hi there. Thanks for raising all the issues and providing your feedback.

I've decided to switch my exif library to the more actively maintained https://www.npmjs.com/package/exif-parser, but I have found an issue with the way they parse dates as well, and have [submitted a pull request](https://github.com/dylansmith/exif-parser/commit/4b3681e78616672b0b12de3a2e26244de68d3d5b) to fix this.

Hopefully I can get all these issues resolved soon!
